prow/cmd/deck/static/script.js

```diff
@@ -216,7 +216,7 @@ function createRerunCell(modal, rerun_command, prowjob) {
     a.title = "Show instructions for rerunning this job.";
     a.onclick = function() {
         modal.style.display = "block";
-        rerun_command.textContent = "kubectl create -f \"" + url + "\"";
+        rerun_command.innerHTML = "kubectl create -f \"<a href='" + url + "'>" + url + "\"";
```

Comment:
Closing tag?

Reply:
pfft [we don't need no stinking close tags](https://google.github.io/styleguide/htmlcssguide.html#Document_Type) /s
I'll fix that, heh.

Reply:
done.